src/test/java/com/reservation/attendance/domain/AttendanceTest.java

```diff
@@ -0,0 +1,98 @@
+package com.reservation.attendance.domain;
+
+
+import com.reservation.attendance.domain.exception.AttendanceAlreadyCancelledException;
+import com.reservation.attendance.domain.exception.DuplicateAttendanceException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+
+class AttendanceTest {
+
+    AttendanceId attendanceId;
+    ClientId clientId;
+    SessionOccurrenceId sessionId;
+    Attendance attendance;
+
+    @BeforeEach
+    void setUp() {
+        attendanceId = new AttendanceId(UUID.randomUUID());
+        clientId = new ClientId(UUID.randomUUID());
+        sessionId = new SessionOccurrenceId(UUID.randomUUID());
+        attendance = new Attendance(attendanceId, clientId, sessionId);
+    }
```

Comment:
Tutaj bym wymusił użycie metod z VO -> next() + do wywalenia pewnie import UUID z klasy testowej

```suggestion
    @BeforeEach
    void setUp() {
        attendanceId = AttendanceId.next();
        clientId = ClientId.next();
        sessionId = SessionOccurrence.next();
        attendance = new Attendance(attendanceId, clientId, sessionId);
    }
```

Reply:
Wydawało mi się że to dodawałem ale jednak nie było.